.neutrinorc.js

```diff
@@ -58,6 +43,7 @@ module.exports = {
             "class-methods-use-this": 0,
             "camelcase": 0,
             "global-require": 0,
+            "react/destructuring-assignment": 0,
```

Comment:
Might be worth sorting these alphabetically at some point? (And and also fixing them gradually in separate PRs)